src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -102,26 +104,29 @@ def _webhook_event_data(
 @instrumented_task(name="sentry.sentry_apps.tasks.sentry_apps.send_alert_event", **TASK_OPTIONS)
 @retry_decorator
 def send_alert_event(
-    event: Event | GroupEvent,
+    instance_id: str,
+    group_id: int,
+    occurrence_id: str | None,
     rule: str,
     sentry_app_id: int,
     additional_payload_key: str | None = None,
     additional_payload: Mapping[str, Any] | None = None,
 ) -> None:
     """
     When an incident alert is triggered, send incident data to the SentryApp's webhook.
-    :param event: The `Event` for which to build a payload.
+    :param instance_id: The event id to reference when pulling data from nodestore
+    :param group_id: The Group for the event
+    :param occurrence_id: Corresponding IssueOccurrence for the event
     :param rule: The AlertRule that was triggered.
     :param sentry_app_id: The SentryApp to notify.
     :param additional_payload_key: The key used to attach additional data to the webhook payload
     :param additional_payload: The extra data attached to the payload body at the key specified by `additional_payload_key`.
     :return:
     """
-    group = event.group
+    group = Group.objects.get_from_cache(id=group_id)
```

Comment:
You'll need to go down this path based on the parameter values you get. There will be messages in the queues that don't have `group_id`.